### PR TITLE
CHECK: These changes look wrong. Check that they really are correct

### PR DESCRIPTION
**Pull request by magicus**
Do not merge until all changes are confirmed.

### CODE REVIEW

File: common/src/main/java/com/wynntils/mc/event/ContainerSetContentEvent.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import net.minecraftforge.eventbus.api.Event;
 
 public abstract class ContainerSetContentEvent extends Event {
-    private final List<ItemStack> items;
```
Comment (magicus) [inline common/src/main/java/com/wynntils/mc/event/ContainerSetContentEvent.java:13]: Okay, I understand and accept these changes. They are correct.

File: common/src/main/java/com/wynntils/gui/screens/settings/elements/EnumConfigOptionElement.java [hunk 2]
```diff
@@ -84,8 +84,6 @@ public boolean mouseClicked(double mouseX, double mouseY, int button) {
             return false;
         }
 
-        McUtils.playSound(SoundEvents.UI_BUTTON_CLICK.value());
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/gui/screens/settings/elements/EnumConfigOptionElement.java:87]: It was duplicated. If you look below, we call it.

File: common/src/main/java/com/wynntils/gui/widgets/TextInputBoxWidget.java [hunk 3]
```diff
@@ -254,7 +254,7 @@ public boolean isFocused() {
     }
 
     @Override
-    public void updateWidgetNarration(NarrationElementOutput narrationElementOutput) {}
+    protected void updateWidgetNarration(NarrationElementOutput narrationElementOutput) {}
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/gui/widgets/TextInputBoxWidget.java]: The method is now protected. We should change all public cases to protected.

Comment (magicus) [reply common/src/main/java/com/wynntils/gui/widgets/TextInputBoxWidget.java]: We have this in lots of other places, which are public. I suggest we revert these two odd ones, and then make a separate change to make them protected. (And it is always legal to widen scope, so it is more an issue of style)

File: common/src/main/java/com/wynntils/gui/screens/settings/elements/EnumConfigOptionElement.java (hunk 2 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/gui/screens/settings/elements/EnumConfigOptionElement.java:87]: So this is a bug fix? Can we do that separately directly to main?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/gui/screens/settings/elements/EnumConfigOptionElement.java:87]: Sure.

**Comment by kristofbolyai** [comment]
@magicus I think this PR served it's purpose. Can you merge the container event changes? I opened a PR for the sound removal on main.

**Pull request merged**